Ecosystem Partners/Marketplace Guides/getting-started-with-waratek-locker.md

```diff
@@ -0,0 +1,153 @@
+{{{ 
+  "title": "Getting Started With Waratek Locker – Runtime Application Self Protection", 
+  "date": "10-21-2015", 
+  "author": "David Egan", 
+  "attachments": [], 
+  "contentIsHTML": false
+}}}
+
+### ![Waratek Logo](http://cdn.aws.waratek.com/wp-content/themes/waratek/images/logo.png)
+
+###Partner Profile
+
+Waratek, a pioneer in Runtime Application Self Protection, secures applications from logic, network and vulnerability exploits without code changes, agents or hardware devices, placing security within your own control. This enables applications to be run on a public cloud while retaining a level of protection normally only available in the enterprise data center.
+
+In 2015 the RSA Conference’s Innovation Sandbox awarded Waratek the title of Most Innovative Company saying it was *an inventive solution to a ‘massive problem’.*
+
+###Contact Waratek
+
+**Waratek Sales and Support:**
+
+* Email support – [support@waratek.com](mailto:support@waratek.com)
+
+* Support Site – [https://support.waratek.com](https://support.waratek.com/). 
+
+* Sales and Marketing – [sales@waratek.com](mailto:sales@waratek.com)
+
```

Comment:
get rid of the dashes and just use the text as the link.  i.e. change:
Email support – [support@waratek.com](mailto:support@waratek.com)
to
[Email support](mailto:support@waratek.com)